sorc/ncep_post.fd/MDL2P.f

```diff
@@ -3526,7 +3554,11 @@ SUBROUTINE MDL2P(iostatusD3D)
                    ELSE
                      IMOIS = 3
                    END IF
+                 IF(TSL(I,J)<spval.and.TPRS(I,J,LUHI)<spval.and.TDSL(I,J)<spval)THEN
```

Comment:
@JesseMeng-NOAA 
Add checks for PSL and QSL too

Reply:
will do